app/partials/footer.html

```diff
@@ -1,3 +1,58 @@
-<div class="footer">
-  <p>♥ from the Yeoman team</p>
+<div class="row footer">
+  <div class="col-lg-2 col-md-offset-2 list">
+     <ul style="list-style-type:none">
```

Comment:
In general, avoid using inline style attribute. It's not maintainable.
Also, if you find trying to style multiple elements with the same style, as in this case, it's better to create a clase to unstyle the lists.
In this specific case, better use bootstrap helpers, take a look at http://getbootstrap.com/css/#unstyled